tests/benchmarks/CompiledCodeBenchmarks/MicroPerf/Equality.fs

```diff
@@ -19,6 +19,23 @@ type EqualityBenchmarks() =
         Array.init 1000 id
         |> Array.groupBy (fun n -> SomeStruct(n % 7))
 
+    [<Benchmark>]
+    member _.ArrayDistinct() =
+        Array.init 1000 id
```

Comment:
All those benchmarks include array initialisation, it should be just done once per whole benchmark suite.

Reply:
Yes, good point